unstructured/file_utils/filetype.py

```diff
@@ -8,6 +8,8 @@
 from functools import wraps
 from typing import IO, TYPE_CHECKING, Callable, List, Optional
 
+from unstructured_inference.inference.layoutelement import LocationlessLayoutElement
```

Comment:
I confirmed this causes `test_unstructured_ingest/unit/test_interfaces.py::test_process_file_fields_include_default` to fail.

Reply:
Due to the observation in the comment above, closing in favor of https://github.com/Unstructured-IO/unstructured/pull/913